ush/set_rrfs_config_general.sh

```diff
@@ -122,14 +123,14 @@ START_TIME_CONVENTIONAL_SPINUP="00:40:00"
 START_TIME_CONVENTIONAL="00:50:00"
 START_TIME_NSSLMOSIAC="00:55:00"
 START_TIME_LIGHTNINGNC="00:55:00"
-START_TIME_PROCSMOKE="00:55:00"
+START_TIME_PROCSMOKE="00:50:00"
 
 if [[ ${PREDEF_GRID_NAME} == "RRFS_CONUS_3km" ]] ; then 
   DT_ATMOS=36
   ADDNL_OUTPUT_GRIDS=()
   TILE_LABELS="CONUS REGIONS"
   TILE_SETS="full NE,NC,NW,SE,SC,SW"
-  CCPP_PHYS_SUITE="FV3_HRRR"
+  CCPP_PHYS_SUITE="FV3_HRRR_gf"
```

Comment:
We still need to manually copy this CCPP SDF file, right?

Reply:
Yes, that is the way to be able to compile the code but I hope this can be fixed soon.